.github/workflows/ci-linux.yml

```diff
@@ -32,5 +32,5 @@ jobs:
     - name: Upload Flatpak
       uses: actions/upload-artifact@v3
       with:
-        name: obliteration-linux64
+        name: obliteration-linux-x64
```

Comment:
Better to use `x86_64`, `x86-64` or `amd64` because x64 is not an official architecture name on Linux world.

Reply:
This is a hate crime.